libs/SmartSync/SmartSync/Classes/Manager/SFSmartSyncSyncManager.m

```diff
@@ -96,6 +101,7 @@ + (instancetype)sharedInstanceForStore:(SFSmartStore *)store {
             syncMgr = [[self alloc] initWithStore:store];
             syncMgrList[key] = syncMgr;
         }
+        [[SalesforceSDKManager sharedManager] registerAppFeature:kSFAppFeatureSmartSync];
```

Comment:
Indeed - I reviewed too quickly (but the comment for the Android PR still stands though: https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/1279)